### DISCUSSION TITLE
Modified to use BLE? Pico has Bluetooth and Wondered if this could transmit over bluetooth?

### DISCUSSION THREAD

**Discussion by nikaskeba** [OrangeFox86/DreamPicoPort#116] [Ideas]
Adding bluetooth option to the build

**Comment by Tails86** [comment]
Sorry for the extremely late reply. Yes, possible. Maybe someday I might get to it, or I could provide guidance if someone wanted to tackle it.

**Reply by Tails86** [reply]
I take it back - I didn't realize the wireless module needed PIO resource. That makes it incompatible with this project, unfortunately 😞
